### PR TITLE
DoNotEnforce policy assignment - Tagging on CFT Sandbox

### PR DESCRIPTION
**Pull request by Jenniferstrej**
### description ###
DoNotEnforce policy assignment for tagging policy on sandbox CFT environment, including some exceptions on AKS related scopes.

Once we finish the tagging work we will switch "DoNotEnforce" to "Default" enforcementMode.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "properties": {
+      "metadata": {
+        "assignedBy": "PlatOps"
+      },
+      "description": "Policy to check required tags",
+      "displayName": "HMCTS Tagging - CFT SBOX",
+      "enforcementMode": "DoNotEnforce",
```
Comment (Jenniferstrej) [inline assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json:8]: Once we finish the tagging work we will switch "DoNotEnforce" to "Default" enforcementMode

File: assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      "nonComplianceMessages": [
+        {
+            "message": "Resource doesn't meet the tagging requirements. See https://github.com/hmcts/azure-policy/blob/master/policies/tagging/README.md"
+        }
+      ],
+      "notScopes": [
```
Comment (Jenniferstrej) [inline assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json:14]: exceptions for AKS related resources that are created automatically and we can't embed tags on them.

File: assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        "/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/resourceGroups/cft-sbox-00-aks-node-rg",
+        "/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/resourceGroups/cft-sbox-01-aks-node-rg",
+        "/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/resourceGroups/sbox-00-rg",
```
Comment (thomast1906) [inline assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json:17]: typo on the resource groups, `cft-sbox` required

**Comment by thomast1906** [comment]
Will message you a URL, there is already a policy set in this subscription. You probably want to remove this prior to applying this one

File: assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json (hunk 3 above)
Comment (thomast1906) [reply assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json:17]: sbox-00-rg is something old, this ignore is in relation to the aks resources that we deploy via https://github.com/hmcts/aks-cft-deploy - there was an issue when Blue implemented this with AKS and its tags. 

Do you want to test this without these exceptions to see if the tags/policy shows correctly?

Comment (Jenniferstrej) [reply assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json:17]: @thomast1906 - thanks, I'll add `cft-sbox-00-rg` to the list.

There is also a resource group `sbox-00-rg`. Are we not supposed to include https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/resourceGroups/sbox-00-rg/overview this one?

Comment (Jenniferstrej) [reply assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json:17]: @thomast1906 - I looked for `cft-sbox-00-rg` and 01 and didn't find any non-compliant resource, but I'll add the exemption for consistency.

**Pull request closed without merging**